src/main/java/org/kuali/kfs/coa/businessobject/lookup/KualiAccountLookupableHelperServiceImpl.java

```diff
@@ -36,6 +38,7 @@
 import org.kuali.kfs.krad.bo.BusinessObject;
```

Comment:
If we're still keeping this overlay, the BusinessObject and PermissionService imports could potentially be removed since I don't think they're being used by this class.